github/ci/services/loki/secrets/testing/loki-config/loki.yaml

```diff
@@ -0,0 +1,44 @@
+auth_enabled: false
+chunk_store_config:
+  max_look_back_period: 0s
+compactor:
+  shared_store: filesystem
+  working_directory: /data/loki/boltdb-shipper-compactor
+ingester:
+  chunk_block_size: 262144
+  chunk_idle_period: 3m
+  chunk_retain_period: 1m
+  lifecycler:
+    ring:
+      kvstore:
+        store: inmemory
+      replication_factor: 1
+  max_transfer_retries: 0
+  wal:
+    dir: /data/loki/wal
+limits_config:
```

Comment:
We haven't yet configured retention here, right? Are there defaults or would we be storing everything forever with this configuration? IIUC the [docs](https://grafana.com/docs/loki/latest/operations/storage/retention/#retention-configuration) correctly you explicitly need to activate this?

Reply:
Good point Daniel, I have made an update to the configuration to enable retention and to retain the logs for 2 weeks.

Reply:
Great, thanks!